lib/data/services/auth/sp_service.dart

```diff
@@ -1,13 +1,23 @@
 import 'package:shared_preferences/shared_preferences.dart';
 
+import '../../../ui/utils/constant.dart';
```

Comment:
Why do you need `SharedPreferences` ? `firebaseAuth` should be sufficient

Reply:
Done